### DISCUSSION TITLE
Stratification map

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2029] [Q&A] [answered]
Hello

Can parameters be sent to the map so that it stratifies by state?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz It might be best if you could provide a link to an existing indicator page with a map, and then describe how you would like it to change?

But my hunch is that you are asking about changing the boundaries used in the map. If so, yes this is possible. The boundaries used in the maps are determined by the GeoJSON file that you provide. So, if you want to use different boundaries, you will need to update the GeoJSON file.

**Comment by SandraEliz** [comment]
Hello

What is intended to be done is that when changing the federative entity, only that entity is displayed, for example:
https://prep-odsmexico.github.io/site/11-1-1/

If it is United Mexican States

![image](https://github.com/open-sdg/open-sdg/assets/117297044/c43d3d57-171b-4834-874a-d5bc606a018b)

If you move to Aguascalientes

![image](https://github.com/open-sdg/open-sdg/assets/117297044/de9a717d-525b-4157-9117-3125d01f4d4d)



Thank you

**Reply by sarah-beasley** [reply, marked as answer]
Hi Sandra, the functionality is that when you click on an area of the map, it zooms to that area and there is no other effect.

**Reply by SandraEliz** [reply]
Thanks for the answer
